src/index.js

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import 'bootstrap/dist/css/bootstrap.min.css';
+//import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
do you really want to bail on bootstrap?

Reply:
I think so after reviewing the rest of the PR.  You should delete this line instead of commenting out though.